distribution/helm/vector-aggregator/templates/configmap.yaml

```diff
@@ -11,6 +11,12 @@ data:
   managed.toml: |
     {{- include "libvector.vectorConfigHeader" . | nindent 4 -}}
 
+    {{- with .Values.vectorApi }}
+    {{- if .enabled }}
+    {{- tuple "api" . | include "libvector.vectorApiConfig" | nindent 4 -}}
+    {{- end }}
+    {{- end }}
+
```

Comment:
Vector API is not a component, and should not go through the component template. Instead, add it to `libvector.vectorConfigHeader`, and just template it directly, like data dir.

Reply:
@MOZGIII okay) i see.